modules/rpm-ostree/rpm-ostree.sh

```diff
@@ -15,13 +15,15 @@ fi
 get_yaml_array INSTALL '.install[]' "$1"
 if [[ ${#INSTALL[@]} -gt 0 ]]; then
     echo "Installing RPMs"
-    echo "Installing: ${INSTALL[*]}"
-    rpm-ostree install "${INSTALL[@]}"
+    INSTALL_STR=$(echo "${INSTALL[*]}" | tr -d '\n')
```

Comment:
I don't get this error (which this commit didn't fix). Running `rpm-ostree install "python3-pip" "libadwaita"` works, but running `rpm-ostree install $INSTALL_STR` where `$INSTALL_STR` is `"python3-pip" "libadwaita"` doesn't. 

Iterating the array could fix it, but this approach did work in [the previous versions](https://github.com/ublue-os/startingpoint/blob/d04ad0f78c2b3a7c32cc4c75bafff0f267535af4/scripts/build.sh#L59C29-L59C29) where the yaml array was output as yaml and not json.

Or maybe it was directly a shell array hmm, let me...

Reply:
That fixed it, now there's another error...